site/content/en/docs/installation/_index.md

```diff
@@ -225,6 +225,6 @@ The currently supported features are:
 
 | Feature | Default | Stage | Since | Until |
 |---------|---------|-------|-------|-------|
-| `PartialAdmission` | `false` | Alpha | 0.4 |  |
+| `PartialAdmission` | `true` | Beta | 0.5 |  |
```

Comment:
Should we use the same form as the Kuernetes document?

```suggestion
| `PartialAdmission` | `false` | Alpha | 0.4 | 0.4 |
| `PartialAdmission` | `true` | Beta | 0.5 |  |
```

https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-alpha-or-beta-features

Reply:
done